tools/pip/setup.py

```diff
@@ -156,10 +156,9 @@ def skip_markdown_comments(md):
 
 package_data = {'mxnet': [os.path.join('mxnet', os.path.basename(LIB_PATH[0]))],
                 'dmlc_tracker': []}
-if variant.endswith('MKL'):
-    if platform.system() == 'Darwin':
-        shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/build/install/include'),
-                        os.path.join(CURRENT_DIR, 'mxnet/include/mkldnn'))
+if variant != 'NATIVE':
+    shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/include'),
+                    os.path.join(CURRENT_DIR, 'mxnet/include/mkldnn'))
```

Comment:
there's a flag in mxnet runtime library on whether mkldnn is enabled. instead of relying on variant it will be more robust to switch to that as condition instead.

Reply:
you mean something like this: `mxnet.runtime.Features.is_enabled("MKLDNN")` ?

Reply:
oh bummer I forgot to replace that if condition with runtime.Features

Reply:
https://github.com/apache/incubator-mxnet/pull/19195